funcx_sdk/funcx/sdk/login_manager/whoami.py

```diff
@@ -0,0 +1,84 @@
+from datetime import datetime
+
+from globus_sdk import AuthAPIError
+
+from funcx.sdk.login_manager import LoginManager
+from funcx.sdk.utils.printing import print_table
+
+NOT_LOGGED_IN_MSG = "Unable to retrieve user information. Please log in again."
+
+
+def get_user_info(auth_client, user_id, epoch_auth):
+    """
+    Parse username, name, and last authed time from response
+    """
+    user_info = auth_client.get_identities(ids=user_id)
+    return [
+        user_info["identities"][0]["username"],
+        user_info["identities"][0]["name"],
+        datetime.fromtimestamp(epoch_auth).astimezone().isoformat(),
+    ]
+
+
+def print_whoami_info(linked_identities: bool = False) -> None:
+    """
+    Display information for the currently logged-in user.
+    """
+
+    try:
+        auth_client = LoginManager().get_auth_client()
+    except LookupError:
+        raise ValueError(NOT_LOGGED_IN_MSG)
+
+    whoami_headers = ["Username", "Name", "ID", "Auth Time"]
+
```

Comment:
The `funcx-endpoint list` functionality puts the static-width fields first and the variable-width fields on the right.  This is similar in spirit, for example, to what the "long listing" to `ls` does: put all the static width fields on the left and the variable-width file name on the right.

You might consider remaining congruent with that thought-process.  Consider:
```diff
- whoami_headers = ["Username", "Name", "ID", "Auth Time"]
+ whoami_headers = ["ID", "Auth Time", "Username", "Name"]
```
This could also make the logic below, slightly simpler/more concise.  I'll comment there as well.

Reply:
I'd prefer to stay consistent with the ordering in other globus cli `whoami` commands.